common/src/main/java/com/wynntils/core/features/FeatureRegistry.java

```diff
@@ -67,6 +68,72 @@
 public final class FeatureRegistry {
     private static final List<Feature> FEATURES = new ArrayList<>();
 
+    public static void init() {
```

Comment:
You moved it to make it easier to find?

Reply:
Yes, this is literally the only method we touch in the class.